testsuite/tests/typing-unique/overwriting.ml

```diff
@@ -176,7 +176,7 @@ let disallowed_by_regionality (local_ unique_ r) x =
 Line 3, characters 16-17:
 3 |   let ref = ref r in
                     ^
-Error: This value escapes its region.
+Error: This value is "yielding" but expected to be "unyielding".
```

Comment:
This error message got worse. I'm not sure what to do about it, since the axis on which we fail submoding is produced by the mode solver.

Reply:
Something looks wrong to me here. The code in `modes.ml` prefers to tell you that the locality is wrong rather than the yielding mode. I think you should debug why it isn't doing that.

Reply:
This is probably a mode bug in `Pexp_overwrite` - I'm looking.

Reply:
I pushed a dirty fix, but probably not ideal. Let's discuss in the slack channel wrt uniqueness and overwriting.

Reply:
As discussed offline, overwriting will not be avaialble to the users for some time, and we will re-do the type checking at some point. So the dirty fix is fine. Resolved.